variables.tf

```diff
@@ -2,6 +2,12 @@
 # Account Variables
 ##############################################################################
 
+variable "ibmcloud_api_key" {
+  description = "The IBM Cloud platform API key needed to deploy IAM enabled resources."
```

Comment:
Actually this variable is only used by `terraform-ibm-base-ocp-vpc` and its used to create the containers apikey (if one does not exist already) so perhaps update the description here